### ISSUE TITLE
Ability to disable smoothing effects

### ISSUE THREAD

**Issue by ksvhome** [openlumi/lumimqtt#20]
Some automations, like indicating status of virtual assistant, need instant reaction. It will be nice to implement some option to disable smoothing effects of light. May be add this option in lumimqtt.json file?

**Comment by devbis** [comment]
I implemented it in the last commit. Can you please check the version from github to check if it works with your configuration?

```
pip3 install -U https://github.com/openlumi/lumimqtt/archive/refs/heads/main.zip
```

**Comment by devbis** [comment]
FIxed with 96aadab4363a4ccfca14fb2ec50c69a80f997a55
